src/qp_problems.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+import pandas as pd
+import qpsolvers
+from qpsolvers import solve_qp
+import scipy
+from helper_functions import nearestPD
+from covariance import Covariance
+from constraints import Constraints
+from optimization_data import OptimizationData
+
+
+
+# This class converts an financial optimization problem to a standard quadratic optimization.
```

Comment:
should the name of the class declare its use? e.g. ConvertFinancialtoQuadraticOptimization or something like this? Also the name of the file is simpler to be the same as the name of the class

Reply:
I added the comment for explanation. I am considering GeneralQuadraticProgram, as ConvertFinancialtoQuadraticOptimization starts with a verb and sounds more like a function than a class to me. We can rename maybe Optimization to PorfolioOptimProblem to distinguish with a GeneralQuadraticProgram.